reset-password.py

```diff
@@ -109,4 +109,4 @@ def validate_pin(pin):
         email.dump_to_file(config)
         raise
 
-   
+    print "Password successfully reset for user {}".format(user.name)   
```

Comment:
not sure where this `print` gonna go. Is this supposed to show to anyone? We could have a `logger` (log) maybe?